src/app/components/add-volunteers/add-volunteers.component.ts

```diff
@@ -27,18 +27,14 @@ export class AddVolunteersComponent implements OnInit {
   contentTitle: string;
   contentCategory: any;
   isFileSelected = false;
+  fileErrorSize = false;
```

Comment:
This variable is boolean, but its name hints that it's a number. That's somewhat counterintuitive.
I'd try naming it `isFileValid` or something like this (if I get its meaning right...), and invert the values accordingly.

(Also for add-content.component)